okhttp-tests/src/test/java/com/squareup/okhttp/TestUtil.java

```diff
@@ -15,4 +16,8 @@ private TestUtil() {
     }
     return result;
   }
+
+  public static boolean isEqualCollections(Collection<?> a, Collection<?> b) {
```

Comment:
Damn, I just woke up with thoughts about this problem  of comparing collections, I'll change `List` to the `Set`. 

Yeah, I asked Jake some time ago why he use `LinkedHashSet` (Linked versions of collections in general)  instead of `HashSet`,  he said same thing about determinism. Agree.